gradle/versions.gradle

```diff
@@ -27,7 +27,7 @@ ext.libVersions =
     jackson_annotation: '2.5.0',
     jacoco: '0.7.7.201606060606',
     snakeyaml: '1.12',
-    cassandra: '2.2.8',
+    cassandra: '3.10',
```

Comment:
Whoa there. Do we have an airborne-research-facility compatible module for Cassandra 3.10?